CS.Security/DTO/AdminDto.cs

```diff
@@ -1,7 +1,5 @@
 namespace CS.Security.DTO;
 
-public class AdminDto
-{
-    public Guid Id { get; set; }
-    public string Email { get; set; }
-}
\ No newline at end of file
+public record AdminDto(
```

Comment:
Please find a time to read about records, their difference with classes and how it can make a code more readable and concise. This is not a requirement nor a bug, but generally code with records looks better and works better